docs/interacting-with-fleet-manager.md

```diff
@@ -0,0 +1,99 @@
+# Interacting with Fleet Manager
+
+A user can interact with the Fleet Manager in two main ways:
+* [Directly interacting with the API server](#interacting-directly-with-the-api)
+* [Using the RHOAS CLI](#interacting-using-the-rhoas-cli)
+
+## Interacting directly with the API
+
+The Fleet Manager API requires OIDC Bearer tokens, which use the JWT format,
+for authentication. The token is passed as an `Authorization` HTTP header value.
+
+To be able to perform requests to the Fleet Manager API, you first need
+to get a JWT token from the configured OIDC authentication server, which is
+Red Hat SSO (sso.redhat.com) by default.
+Assuming the default OIDC authentication server is being used, this can be
+performed by interacting with the OCM API.
+This can be easily done interacting with it through the
+[ocm cli](https://github.com/openshift-online/ocm-cli/releases)
+and retrieve OCM tokens using it. To do so:
+1. Login to your desired OCM environment via web console using your Red Hat
+   account credentials. For example, for the OCM production environment, go to
+   https://console.redhat.com and login.
+1. Get your OCM offline token by going to https://console.redhat.com/openshift/token
+1. Login to your desired OCM environment through the OCM CLI by providing the
+   OCM offline token and environment information:
+   ```
+   ocm login --token <ocm-offline-token> --url <ocm-api-url>
```

Comment:
this can also be copied from console.redhat.com/openshift/token, should we mention it too?

Reply:
How to obtain the token is already mentioned in https://github.com/bf2fc6cc711aee1a0c2a/kas-fleet-manager/pull/1532/files#diff-42d65bdb3bcf515e8007d9e838cdf0a0ba480e0917b80f1127ddb5be03e5a283R23.
If you are asking about that "ocm login ..." line being able to be coped from that link I don't think that's needed.